src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -72,7 +77,6 @@ export default function VAFacilityPageV2() {
     sortMethod,
     typeOfCare,
   } = useSelector(state => getFacilityPageV2Info(state), shallowEqual);
-  const [sortType, setSortType] = useState(sortMethod);
```

Comment:
I removed this because the sort method is stored in Redux, and mirroring it in local state seems confusing to me. It also means it's null at unexpected times on the page. It seems easier to make the sort method from Redux be the source of truth.